src/components/SafeListSidebar/SafeList/SafeListItem.tsx

```diff
@@ -85,7 +85,7 @@ const SafeListItem = ({
 
   useEffect(() => {
     if (isCurrentSafe && shouldScrollToSafe) {
-      safeRef?.current?.scrollIntoView({ behavior: 'smooth' })
+      safeRef?.current?.scrollIntoView()
```

Comment:
You can still pass `block: 'center'` for better positioning.

Reply:
But is it not then at the top of the page?

Reply:
`block: 'center'` is supposed to keep the viewport where it is if the item is already visible.
The default is `block: start`, and it will always scroll the page so that the item is on top of it.

Reply:
Need to experiment though, I'm not 100% sure it will be better.

Reply:
I can happily change it. What are your thoughts @liliiaorlenko? Open the sidebar so the currently open Safe is at the top of the list or in the middle?

Reply:
I think In a middle would be the best (to not mess with the order?)

Reply:
I will alter it now. (It doesn't change the order @liliiaorlenko, just the scroll position)